### PR TITLE
Add documentation - Getting Started : Add new Controller

### PR DESCRIPTION
**Pull request by Aschen**

### CODE REVIEW

File: doc/2/guides/getting-started/6-add-new-controller/index.md [hunk 1]
```diff
@@ -4,4 +4,96 @@ type: page
...
+      handler: async request => {
+        return `Hello, ${request.input.args.name}`;
+      }
+    }
+  }
+})
+```
+
+The code above will register a `greeting` controller with a `sayHello` action.  
+This action will use the `name` argument from the request and return a string.
```
Comment (scottinet) [inline doc/2/guides/getting-started/6-add-new-controller/index.md:38]:
```suggestion
This action uses the `name` argument from the request and returns a string.  
```

File: doc/2/guides/getting-started/6-add-new-controller/index.md [hunk 2]
```diff
@@ -4,4 +4,96 @@ type: page
...
+    }
+  }
+})
+```
+
+The code above will register a `greeting` controller with a `sayHello` action.  
+This action will use the `name` argument from the request and return a string.  
+
+::: info
+Kuzzle will generate a default HTTP route of the following format if none are provided:
```
Comment (scottinet) [inline doc/2/guides/getting-started/6-add-new-controller/index.md:41]:
```suggestion
Kuzzle will generate a default HTTP route of the following format if none is provided:   
```

File: doc/2/guides/getting-started/6-add-new-controller/index.md [hunk 3]
```diff
@@ -4,4 +4,96 @@ type: page
...
+{
+  "requestId": "383e016a-447e-4138-aa53-c07c9fd5c923",
+  "status": 200,
+  "error": null,
+  "controller": "greeting",
+  "action": "sayHello",
+  "collection": null,
+  "index": null,
+  "volatile": null,
+  "result": "Hello, undefined"
```
Comment (scottinet) [inline doc/2/guides/getting-started/6-add-new-controller/index.md:76]: So... I guess the controller didn't receive the `name` argument? :thinking:

**Review by scottinet — CHANGES_REQUESTED** [review]

File: doc/2/guides/getting-started/6-add-new-controller/index.md (hunk 3 above)
Comment (Aschen) [reply doc/2/guides/getting-started/6-add-new-controller/index.md:76]: Ouuups :sweat_smile:

**Review by scottinet — APPROVED** [review]

**Review by rolljee — APPROVED** [review]

**Review by xbill82 — APPROVED** [review]

**Pull request merged**